inc/class-wpseo-meta.php

```diff
@@ -944,39 +946,49 @@ public static function keyword_usage( $keyword, $post_id ) {
 			return [];
 		}
 
-		$query = [
-			'meta_query'     => [
-				'relation' => 'OR',
-				[
-					'key'   => '_yoast_wpseo_focuskw',
-					'value' => $keyword,
-				],
-			],
-			'post__not_in'   => [ $post_id ],
-			'fields'         => 'ids',
-			'post_type'      => 'any',
-
-			/*
-			 * We only need to return zero, one or two results:
-			 * - Zero: keyword hasn't been used before
-			 * - One: Keyword has been used once before
-			 * - Two or more: Keyword has been used twice before
-			 */
-			'posts_per_page' => 2,
-		];
+		/**
+		 * @var Indexable_Repository
+		 */
+		$repository = YoastSEO()->classes->get( Indexable_Repository::class );
+
+		$post_ids = $repository->query()
+			->select( 'object_id' )
+			->where( 'primary_focus_keyword', $keyword )
+			->where( 'object_type', 'post' )
+			->where_not_equal( 'object_id', $post_id )
+			->limit( 2 )
```

Comment:
We set `limit( 2 )` to indicate that a keyword has been used twice or more before, correct? The test instructions say "you should see a notification you've used it twice before." Shouldn't this notification say that you've used the keyword twice _or more_ before?

Reply:
Correct, I've updated the test instructions.